proposals/utils/stepper.py

```diff
@@ -0,0 +1,99 @@
+from proposals.models import Proposal
+from django.urls import reverse
+from django.template import loader, Template
+from django.core.exceptions import ImproperlyConfigured
+
+def flatten(lst):
+    if lst == []:
+        return lst
+    first_item = lst[0]
+    rest = lst[1:]
+    if type(first_item) is list:
+        return flatten(first_item) + flatten(rest)
+    return [first_item] + flatten(rest)
+
+
+class renderable:
```

Comment:
```suggestion
class Renderable:
```

Altough, these generic classes should probably live in `main`?